.github/workflows/release-test.yml

```
@@ -0,0 +1,121 @@
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      
+      - name: Set up environment
+        shell: bash
+        run: |
+          if [ "${{ runner.os }}" = "Windows" ]; then
+            choco install make -y
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)